.github/workflows/convert-cci-list.yml

```diff
@@ -0,0 +1,74 @@
+name: Convert CCI List XML to JSON
+
+on:
+  push:
+    branches: ['master']
+
+  # Run this workflow at 00:00 every month
+  schedule:
+    - cron: '0 0 1 * *'
+
+jobs:
+  convert-cci-list:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+          check-latest: true
+          cache: 'yarn'
+
+      - name: Install project dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Prepare environment
+        run: apt update && apt install -y unzip
+
+      - name: Download CCI List
+        run: curl -o U_CCI_List.zip https://dl.dod.cyber.mil/wp-content/uploads/stigs/zip/U_CCI_List.zip && unzip U_CCI_List.zip
+
+      - name: Convert CCI List XML to two JSON files
+        run: yarn workspace @mitre/hdf-converters cciListXml2json /U_CCI_List/U_CCI_List.xml U_CCI_List.nist.json U_CCI_List.defs.json
+
+      - name: Update CciNistMappingData.ts
+        run: |
+          touch tmp.ts
+          echo "export const CCI_TO_NIST: Record<string, string> = " >> tmp.ts
+          cat U_CCI_List.nist.json >> tmp.ts
+          echo ";" >> tmp.ts
+          echo "export const CCI_TO_DEFINITION: Record<string, string> = " >> tmp.ts
+          cat U_CCI_List.defs.json >> tmp.ts
+          echo ";" >> tmp.ts
+          cat tmp.ts > libs/hdf-converters/src/mappings/CciNistMappingData.ts
+          rm tmp.ts
+
+      - name: Commit changes to CciNistMappingData.ts
+        run: |
+          git config --local user.email "saf@groups.mitre.org"
+          git config --local user.name "MITRE SAF Automation"
+          git add libs/hdf-converters/src/mappings/CciNistMappingData.ts
+          git commit -sm "Update CCI List to NIST and definition mappings"
+
+      - name: Push changes to repository
+        uses: ad-m/github-push-action@master
+        with:
+          github_token: ${{secrets.GITHUB_TOKEN}}
+          branch: ${{github.ref}}
+  prettier:
```

Comment:
remove this.  if you want to format the file that you generated, do it as another step in the above job to fix specifically that file.  you don't need an action for it either.  just do `npx prettier` - npx will be available because npm is available.

i am very much against automated formatting or linting being committed by ci/cd.